README.md

```diff
@@ -0,0 +1,156 @@
+# Conduit Connnector Google Sheets
+
+### General
+
+The Conduit Connector for [Google Sheets](https://github.com/gopherslab/conduit-connector-google-sheets) provides both source as-well-as destination connector for the Google sheets
+
+### How to build it
+
+Run `make`.
+
+### Testing
+
+Run `make test` to run all the tests.
+
+To run the integration tests under acceptance_test.go, These env variables need to be set:
+`CONDUIT_GOOGLE_CREDENTIAL_JSON`: this env variable should contain the google service account's credentials JSON
+`CONDUIT_GOOGLE_TOKEN_JSON`: this env variable should contain the oauth2 token JSON containing at least `refresh_token`.
+`CONDUIT_GOOGLE_SHEET_URL`: the Google sheet URL, used to get the spreadsheet id and sheet id
+`CONDUIT_GOOGLE_SHEET_NAME`: the name of the target sheet, this is required to be able to write to the sheet
```

Comment:
Nitpick: when I go to preview, all of this is shown as a single line.

Reply:
Fixed